src/Compilers/Core/Portable/Compilation/IStatement.cs

```diff
@@ -70,7 +70,7 @@ public interface ISwitchStatement : IStatement
     /// <summary>
     /// Represents a C# case or VB Case statement.
     /// </summary>
-    public interface ICase
+    public interface ICase : IStatement
```

Comment:
I will address this in another PR, since this one has been dragging for too long, I'd like to merge this to as soon as possible to unblock any other work related to IOperation.